src/components/common/header/HelpNav.tsx

```diff
@@ -32,7 +36,7 @@ function HelpNav({ className, showHelpCard, setShowHelpCard, serverInfo, fetchin
             name: 'Chat with support',
             link: 'https://discord.devtron.ai/',
             icon: Chat,
-            showSeparator: true,
+            showSeparator: isEnterprise ? false : true,
```

Comment:
can be simplified by using `!isEnterprise` directly

Reply:
Done